lib/connect.ts

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-var axios = require("axios");
-var csvParse = require("papaparse");
-var sha256 = require("crypto-js/sha256");
-var querystring = require("querystring");
-var utils = require("./utils");
+import axios from "axios";
+import csvParse from "papaparse";
+import sha256 from "crypto-js/sha256";
+const querystring = require("querystring");
```

Comment:
Can be converted to an import statement.
Will need to install `@types/node` as a devDependency.